index.html

```diff
@@ -9,14 +9,14 @@
     <title>Garvin Chimone</title>
 </head>
 <body>
-    <nav>
-        <div class="title"><h2>My Logo</h2></div>
+    <header>
+        <div class="title"><a href="index.html"><h2>My Logo</h2></a></div>
         <div class="burger-menu">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
         </div>
-    </nav>
+    </header>
     
     <section class="headline">
```

Comment:
- [ ] Kindly consider to apply display:flex in this section as it is a requirement in the guidelines.